docs/CONTRIBUTING.md

```diff
@@ -1,3 +1,11 @@
+# Contribution Documentation for Developers & Translators
+
+### Table of Contents
```

Comment:
even though codacy says to set this as h2, I think it would be better to replace all except this as h2.
Basically: 
- keep this table of contents as h2 (###)
- replace the other entries (For coder/developer, Installing...) with h1 (##)